### ISSUE TITLE
perspective-cli has no output when run

### ISSUE THREAD

**Issue by mhkeller** [finos/perspective#2085]
### Steps to Reproduce:
 1. `npm install -g @finos/perspective`
 2. `perspecitve host -o file.csv

### Expected Result:
It opens a perspective viewer

### Actual Result:
Nothing happens

### Environment:
macOs 12.6.1
node v18.12.1

### Additional Context:

https://user-images.githubusercontent.com/498744/213074037-a0987480-36b7-48bb-bcbd-cd549fdda56e.mov

**Comment by texodus** [comment]
@mhkeller Thanks for the report @mhkeller - this was a dumb regression by me in #2079.  I have a fix for this in #2086 and it will be released later today in v1.9.2

**Comment by texodus** [comment]
@mhkeller v1.9.2 is out now and should be fixed, please let us know if you have further issues with it

**Comment by mhkeller** [comment]
Cool that works! Some small usability points:

1. It loads saying `Untitled` as the name. It would be nice if I fed it a file name, it displayed that
2. It would be nice if it launched with the sidebar open so I could start filtering, grouping etc.
3. Is there any way to get chrome to not have the "this is being controlled by automation software" banner? 

<img width="1208" alt="Screen Shot 2023-01-18 at 4 08 53 PM" src="https://user-images.githubusercontent.com/498744/213295825-f0433d46-4815-42ee-bc83-1c1ba9a0cc08.png">

**Comment by mhkeller** [comment]
I see you have the same problem where if you click on the chromium icon in the doc you get a new window launched. I ran into that here https://github.com/mhkeller/plot/issues/6. I tried to get it fixed in chromium but I was told it was too niche.

The fix I put in to plot is I listen for a new window and immediately close it: https://github.com/mhkeller/plot/blob/07b4b05673d5793e34e20b82dbcc8e19a248af72/utils/launchContextPage.js#L31
